### PR TITLE
Add firebase realtime db

### PR DESCRIPTION
**Pull request by ajay-sreeram**
* Adds a Firebase real time database loader which uses the firebase-admin
* Adds Readme, requirements for this loader

### CODE REVIEW

**Comment by ajay-sreeram** [comment]
Hi @jerryjliu , I kindly request your review of my pull request on adding support to realtime db. Your feedback would be greatly appreciated. Thank you!

File: llama_hub/firebase_realtimedb/base.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+"""Firebase Realtime Database Loader."""
+
+from typing import Dict, List, Optional
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+import firebase_admin
```
Comment (jerryjliu) [inline llama_hub/firebase_realtimedb/base.py:7]: make lazy import in `__init__` and `load_data` (see other loaders for examples)

**Review by jerryjliu — COMMENTED** [review]
thanks! one comment

File: llama_hub/firebase_realtimedb/base.py (hunk 1 above)
Comment (ajay-sreeram) [reply llama_hub/firebase_realtimedb/base.py]: Sure @jerryjliu, thank you for your feedback! I have made the required changes and implemented lazy import in both the __init__ and load_data functions. Please take a look at the updated code. Your further review would be greatly appreciated. Thanks again!

**Pull request merged**